src/components/driverCard.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DriverCard = ({ name, rating, img, car }) => {
+  const roundedRating = Math.round(rating)
+  
+    return (
+    <div className="driver-card">
+      <div className="driver-image">
+        <img src={img} alt={`${name}'s profile`} />
+      </div>
+      <div className="driver-info">
+        <h2>{name}</h2>
+        <div className="driver-rating">
+          {Array.from({ length: 5 }).map((_, index) => (
```

Comment:
Maureen, here, since you received the `rating`, you previously built a component `Rating.jsx` that you could've used instead of repeating the process.